packages/client/src/App.tsx

```diff
@@ -1,21 +1,27 @@
-// import { useEffect } from 'react';
-// import React from 'react';
-import { Route, Routes } from 'react-router-dom'
+import { Route, Routes, useLocation } from 'react-router-dom'
 import { MainLayout } from './components/MainLayout'
 import { Main } from './pages/Main'
+import { Login } from './pages/Login'
+import { Registration } from './pages/Registration'
+import { Profile } from './pages/Profile'
+import { Leaderboard } from './pages/Leaderboard'
+import { NotFound } from './pages/NotFound'
+import { Forum } from './pages/Forum'
 
 import './App.css'
 
 function App() {
+  const pathname = useLocation().pathname
```

Comment:
Можно деструктурировать
```suggestion
  const { pathname } = useLocation()
```